src/router.js

```diff
@@ -104,7 +104,7 @@ const routes = [
     },
   },
   {
-    path: '/stats/:seasonId?',
+    path: '/stats/:seasonId',
```

Comment:
this should have the `?` this way you can see the stats page with or without a seasonId